pkg/webhooks/webhooks.go

```diff
@@ -208,3 +210,16 @@ func HealthProbe(ctx context.Context) healthz.Checker {
 		return nil
 	}
 }
+
+func ValidateConversionEnabled(ctx context.Context, kubeclient client.Client) {
+	// allow context to exist longer than cache sync timeout
+	listCtx, cancel := context.WithTimeout(ctx, 130*time.Second)
```

Comment:
nice! Can we ref to where the cache sync timeout is defined? Is this configurable or will this always be 2m?

Reply:
It's configurable here: https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/controller/controller.go#L49

I don't see us changing this default anywhere in the repo.